src/scripts/items/ItemHandler.ts

```diff
@@ -23,6 +23,26 @@ class ItemHandler {
         return result == undefined ? true : result;
     }
 
+    public static useAllBattleItems() {
+        GameHelper.chunk(6, Object.keys(ItemList).filter(i => ItemList[i].constructor.name == 'BattleItem')).forEach((item,index) => {
```

Comment:
I don't think we need the `GameHelper.chunk()`. The filter should leave us with only the battle items. And this will make it harder to add more stuff (which is probs more relevant for flutes).